rosa-hypershift/rosa-hosted-wrapper.py

```diff
@@ -1103,6 +1109,20 @@ def _watcher(rosa_cmnd, my_path, cluster_name_seed, cluster_count, delay, my_uui
         all_clusters_installed.notify_all()
     logging.info('Watcher terminated')
 
+def _get_git_repo_details():
+    # Get the Git remote URL for the current working directory
+    old_wd = os.getcwd()
+    current_wd = os.path.dirname(os.path.abspath(__file__))
+    os.chdir(current_wd)
```

Comment:
@morenod This tackles the issue: even if the user triggers the wrapper script outside the GitHub repo.